client/components/cart-summary.jsx

```diff
@@ -6,6 +6,10 @@ class CartSummary extends React.Component {
     const cartList = this.props.cart.map(product => {
       return <CartSummaryItem key={product.cartItemId} product={product} />;
     });
+    let itemTotal = 0;
+    this.props.cart.map(product => {
+      itemTotal += product.price;
+    });
```

Comment:
If you are going to ignore the array that `.map()` returns, use `.forEach()` instead.